README.md

```diff
@@ -1,51 +1,66 @@
-# loopback4-extension-grpc
+# gRPC Extension for LoopBack 4
 
 [![Join the chat at https://gitter.im/strongloop/loopback4-extension-grpc](https://badges.gitter.im/strongloop/loopback4-extension-grpc.svg)](https://gitter.im/strongloop/loopback4-extension-grpc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-gRPC Extension for LoopBack 4
-
-## Summary
-A gRPC extension for LoopBack Next.
-
 ## Overview
-TODO.
+The `@loopback/grpc` component enables LoopBack 4 as a Grpc Server. Also it provides with a @grpc decorator to define your RPC Method implementations from your Application Controllers.
```

Comment:
Please add a url for gRPC. Change @grpc to `@grpc`.

Reply:
Sure